editors/vscode/src/browser.ts

```diff
@@ -46,6 +46,15 @@ function startClient(context: vscode.ExtensionContext) {
                     return;
                 });
                 //client.onNotification(serverStatus, (params) => setServerStatus(params, statusBar));
+
+                vscode.workspace.onDidChangeConfiguration(async (ev) => {
+                    if (ev.affectsConfiguration("slint")) {
+                        await client.sendNotification("workspace/didChangeConfiguration", { settings: "" });
+                        if (previewUrl) {
```

Comment:
Ahh, this works because of the implicit conversion of string to bool, right?

Reply:
not sure what you mean.  but yeah, the idea is that if previewUrl is defined (and not empty),  we should refresh the preview. otherwise not.